distribution/docs/src/main/resources/content/_highAvailability/high-availability-supported-features.adoc

```diff
@@ -0,0 +1,190 @@
+:type: appendix
+:status: published
+:parent: High Availability Supported Features
+:children: none
+:order: 00
+:summary: High Availability Supported Features
+
+This is the list of ${branding} features available in a Highly Available Cluster:
+
+.High Availability Features
```

Comment:
❗️ Kinda of concerning that we list all features here. Should we simply refer to a profile file that is installed? And if we don't, should we install one for DDF?

Reply:
I agree that this will become a maintenance problem in the future. Reference to the profile file sounds good to me.

Reply:
+1 reference the profile especially since our feature configuration is arbitrary to us and we don't want to be coupled to it